File: 1/a.py
import sys

lefts = []
rights = []

for line in sys.stdin:
    bits = line.strip().split('   ')
    lefts.append(int(bits[0]))
    rights.append(int(bits[1]))

lefts.sort()
rights.sort()

print(sum(abs(x[0]-x[1]) for x in zip(lefts, rights)))

File: 1/a.go
package main

import (
	"bufio"
	"fmt"
	"os"
	"slices"
	"strconv"
	"strings"
)

func MustInt(s string) int {
	rv, err := strconv.Atoi(s)
	if err != nil {
		panic(err)
	}
	return rv
}

func main() {
	var left, right []int
	scanner := bufio.NewScanner(os.Stdin)
	for scanner.Scan() {
		bits := strings.Split(scanner.Text(), "   ")
		left = append(left, MustInt(bits[0]))
		right = append(right, MustInt(bits[1]))
	}
	slices.Sort(left)
	slices.Sort(right)
	s := 0
	for i, l := range left {
		d := right[i] - l
		if d < 0 {
			d *= -1
		}
		s += d
	}
	fmt.Println(s)
}
